### PR TITLE
Feature/bg design

### PR DESCRIPTION
**Pull request by Satyam1RP**
Background Design  Component Code is Done and ready for review.

### CODE REVIEW

**Comment by mathrik** [comment]
I pulled down your branch, but the responsive behavior looked different than what you demo'd earlier today. Would you mind double checking your commits? I'd also appreciate it if you pasted the specs in a comment here.

**Comment by Satyam1RP** [comment]
Hi @mathrik  here is the assignment file: 
[Assignment-09-07-2024.docx](https://github.com/user-attachments/files/16190171/Assignment-09-07-2024.docx)

I put the specifications from Assignment No. 8 to the scss file, and you may review all of them there as well.

Yes, the appearance differs from the demo I showed you; I updated the design and markup.
For this work, I set the first theme as the default and added three different versions of the same design, each with its own markup.

Required specs:  
image
 

here is the snapshot for mobile and desktop variation: 
Mobile: 
image

Desktop : 
image


CC: @jvaishnavrp @Lalit-gour1

File: Assets/scss/components/bg-design.scss [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            padding: toRem(10) toRem(30);
+            margin-inline-end: toRem(300);
+            @include min-screen(sm) {
+                margin-inline-end: 0;
+            }
+        }
+       
+    }
+
+    &-bottom{
```
Comment (jwetmore-hoodoo) [inline Assets/scss/components/bg-design.scss:43]: Could potentially use an ::after pseudo-element instead, so you don't need to put the empty div in the HTML.

**Comment by mathrik** [comment]
Screenshot 2024-07-12 at 3 21 07 PM
This is what I'm seeing when I build your code locally.

**Comment by mathrik** [comment]
Screenshot 2024-07-12 at 3 21 07 PM
This is what I'm seeing when I build your code locally.

**Comment by Satyam1RP** [comment]
@jwetmore-hoodoo Yes i used ::after now and removed blank div. 
@mathrik  can you take a look again I made a few changes on the last commit.

CC: @jvaishnavrp

File: Assets/scss/base/_var.scss [hunk 2]
```diff
@@ -27,8 +27,11 @@ $grey80: #808080;
 $darkestgrey: #352f2f;
 
 $wine: #3f2222;
+<<<<<<< HEAD
+=======
 $color-black-transparency: rgba(0, 0, 0, .8);
 $right-arrow: '\2192';
+>>>>>>> e78706feb29f743112caee8dcf4dc391612b9022
```
Comment (jwetmore-hoodoo) [inline Assets/scss/base/_var.scss:34]: Git conflicts were committed.

Comment (Satyam1RP) [reply Assets/scss/base/_var.scss]: resolved.

**Comment by mathrik** [comment]
Screenshot 2024-07-15 at 10 19 22 AM
This is what I see today with the latest on this branch.

**Comment by Satyam1RP** [comment]
hey @mathrik  I updated the code for the large screen can you retake the look?

**Pull request merged**